acceptance/tests/peering/peering_connect_namespaces_test.go

```diff
@@ -162,7 +181,8 @@ func TestPeering_ConnectNamespaces(t *testing.T) {
 			})
 
 			// Ensure the secret is created.
-			retry.Run(t, func(r *retry.R) {
+			timer := &retry.Timer{Timeout: 1 * time.Minute, Wait: 1 * time.Second}
+			retry.RunWith(timer, t, func(r *retry.R) {
```

Comment:
This step could potentially take longer now, because the controller will try to read the service address, and error if for example, a load balancer IP doesn't yet exist. So it could take a couple loops through the controller to get a real address here, so a retry of longer than 7s could be very possible, so I increased the timer

Reply:
aha!! thanks for the clarification!!